gateway/1.4/modules/ROOT/pages/flex-gateway-k8-external-dns.adoc

```diff
@@ -0,0 +1,121 @@
+= Provide a Domain Name for Your Kubernetes Service
+
+Use ExternalDNS to expose your Kubernetes service through a domain name instead of a difficult-to-understand external IP or random host name. 
```

Comment:
```suggestion
Use ExternalDNS to expose your an application or service on your Kubernetes cluster through a domain name instead of a difficult-to-understand external IP or random host name. 
```

Reply:
@ahejeij And I'll commit this if you're ok with it. Thx!